.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1

```

Comment:
settings should be under gitignore.
Not everyone uses eclipse, but the biggest problem will be if someone from your team will use eclipse with different settings - he will hate you for this)